Tests/SwiftGRPCTests/ChannelCrashTests.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ChannelCrashTests.swift
+//  SwiftGRPCTests
+//
+//  Created by Sviatoslav Bulgakov on 06/11/2018.
+//
+
+import XCTest
+@testable import SwiftGRPC
+
+class ChannelCrashProvider: Echo_EchoProvider {
+    func get(request: Echo_EchoRequest, session: Echo_EchoGetSession) throws -> Echo_EchoResponse {
+        return Echo_EchoResponse()
+    }
+    
+    func collect(session: Echo_EchoCollectSession) throws -> Echo_EchoResponse? {
+        return Echo_EchoResponse()
+    }
+    
+    func update(session: Echo_EchoUpdateSession) throws -> ServerStatus? {
+        return .ok
+    }
+    
+    func expand(request: Echo_EchoRequest, session: Echo_EchoExpandSession) throws -> ServerStatus? {
+        let parts = request.text.components(separatedBy: " ")
+        for (i, part) in parts.enumerated() {
+            usleep(500000)
+            var response = Echo_EchoResponse()
+            response.text = "Swift echo expand (\(i)): \(part)"
+            try session.send(response) {
+                if let error = $0 {
+                    print("expand error: \(error)")
+                }
+            }
+        }
+        return .ok
+    }
+}
+
+class ChannelCrashTests: XCTestCase {
+    
```

Comment:
Sorry for noticing this only now, but could you please make this a subclass of https://github.com/grpc/grpc-swift/blob/master/Tests/SwiftGRPCTests/BasicEchoTestCase.swift? That should spare you from all the client/server setup work. Also, please remove this extra newline.